### ISSUE TITLE
[Accessibility]: Clicking the copy query button should maintain focus on the copy button

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#43130]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34188

### Problem description

When using keyboard or mouse, after activating the copy query button on the right of the search box, focus is changed to the search box.

### Expected behavior

The focus should stay on the copy button.

<img width="886" alt="image" src="https://user-images.githubusercontent.com/206864/196552673-bab63e9f-6108-48ff-ac14-de6252fc4e30.png">

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

/cc @lguychard
